pyiron_atomistics/lammps/base.py

```diff
@@ -459,9 +459,11 @@ def collect_output(self):
             ),
             group_name="output",
         )
-
-        if len(self.output.cells) > 0:
-            final_structure = self.get_structure(iteration_step=-1)
```

Comment:
Does that mean the `output.cells` and `get_structure` don't work anymore as well or is this somehow restored by writing the final structure below?

Reply:
It does not work until the output is stored in the HDF5 file, which only happens at the end of `collect_output()`.